syft/generic/pointers/pointer_tensor.py

```diff
@@ -385,12 +384,15 @@ def get(self, user=None, reason: str = "", deregister_ptr: bool = True):
                 method. This defaults to True because the main reason people use
                 this method is to move the tensor from the remote machine to the
                 local one, at which time the pointer has no use.
+            get_copy (bool): Setting get_copy True doesn't destroy remote.
 
         Returns:
             An AbstractTensor object which is the tensor (or chain) that this
             object used to point to #on a remote machine.
         """
-        tensor = ObjectPointer.get(self, user=user, reason=reason, deregister_ptr=deregister_ptr)
+        tensor = ObjectPointer.get(
```

Comment:
BUG: You need to change the serde serialization tests to take into consideration the new parameter.
Also, there is an error here
```
return ObjectRequestMessage(obj_id=obj_id, user=None, reason=proto_msg.reason)
E       TypeError: __init__() missing 1 required positional argument: 'get_copy'
```

Reply:
Done